src/components/layout/layout.js

```diff
@@ -43,6 +44,20 @@ export default function Layout({ crumbLabel, children }) {
         </Breadcrumb>
       ) : null}
       {children}
+      {sources?.length ? (
+          <Row className={styles.sources}>
```

Comment:
Should we give this an ID for anchor linking?

Reply:
I have an id for each li element, that way we can link directly to the specific footnote. They are ordered and display in the same order as in the cms.